uns.go

```diff
@@ -187,61 +210,90 @@ func (c *Cns) Owner(domainName string) (string, error) {
 	return data.Owner.String(), nil
 }
 
-func (c *Cns) IpfsHash(domainName string) (string, error) {
+func (c *Uns) IpfsHash(domainName string) (string, error) {
 	records, err := c.Records(domainName, ipfsKeys)
 	if err != nil {
 		return "", err
 	}
 	return returnFirstNonEmpty(records, ipfsKeys), nil
 }
 
-func (c *Cns) HTTPUrl(domainName string) (string, error) {
+func (c *Uns) HTTPUrl(domainName string) (string, error) {
 	records, err := c.Records(domainName, redirectUrlKeys)
 	if err != nil {
 		return "", err
 	}
 	return returnFirstNonEmpty(records, redirectUrlKeys), nil
 }
 
-func (c *Cns) AllRecords(domainName string) (map[string]string, error) {
-	data, err := c.Data(domainName, []string{})
+func (c *Uns) GetAllKeysFromContractEvents(contract interface {
```

Comment:
If you want to define private method in golang - keep it's first character in lowercase. For example:
```suggestion
func (c *Uns) getAllKeysFromContractEvents(contract interface {
```

Reply:
Please add tests for new method event if it's "private" (not exported)